QUICK_START.md

```
@@ -1,197 +1,153 @@
 
-1. **Upload a PDF**: 
-   - Tap "Choose PDF File" in React Native app
-   - Select a PDF file from your device/simulator
+cd bloodwork_app
+npm install
+npx expo start
 
-2. **Start Analysis**:
-   - Tap "Start Analysis" 
-   - Watch the progress indicator
+	•	Expo shows QR code and simulator options (iOS/Android)
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)